compiler/p/codegen/ControlFlowEvaluator.cpp

```diff
@@ -1693,864 +2037,912 @@ TR::Register *OMR::Power::TreeEvaluator::ifacmpneEvaluator(TR::Node *node, TR::C
    return NULL;
    }
 
-
-// also handles acmpeq in 32-bit mode
-// and also: bcmpeq, scmpeq
-TR::Register *OMR::Power::TreeEvaluator::icmpeqEvaluator(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
One variable on its own line is not the coding style recommended I believed.  And, it is not consistent with existing code either.

Reply:
Using one line per parameter in a function declaration seems to be pretty common (though not ubiquitous) in our code, e.g. [1] [2] [3]. Unfortunately, there's not really any documentation of our recommended coding style in the compiler, as every attempt to decide on a specific style has failed thus far.

However, you're right that the style here doesn't quite match what other code tends to do, as the `)` is on the wrong line and the indentation isn't 6 spaces. I've gone ahead and fixed that.

[1] https://github.com/eclipse/omr/blob/f4798c855f8f73e266ccef403a321c9ce21bbdb0/compiler/p/codegen/ControlFlowEvaluator.cpp#L3524-L3529
[2] https://github.com/eclipse/omr/blob/f4798c855f8f73e266ccef403a321c9ce21bbdb0/compiler/p/codegen/OMRMemoryReference.cpp#L141-L145
[3] https://github.com/eclipse/omr/blob/f4798c855f8f73e266ccef403a321c9ce21bbdb0/compiler/codegen/NodeEvaluation.cpp#L264-L267